lib/redux/middleware/sentry_sdk_middleware.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:redux/redux.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import '../../redux/actions.dart';
+import '../../redux/state/app_state.dart';
+
+class SentrySdkMiddleware extends MiddlewareClass<AppState> {
+
+  @override
+  dynamic call(Store<AppState> store, dynamic action, NextDispatcher next) async {
+    if (action is ApiFailureAction) {
+      Sentry.captureException(action.error);
```

Comment:
Is the information about what action failed all in `action.error`?
Is the stack trace of the action that failed going into the event? `captureException` takes the stack trace too.

Reply:
You are right, i refactored the API client and appropriate action so we have this info. 👍